lib/screens/explore_screens/vouch_for_a_member/interactor/viewmodel/vouch_for_a_member_state.dart

```diff
@@ -0,0 +1,47 @@
+part of 'vouch_for_a_member_bloc.dart';
+
+class VouchForAMemberState extends Equatable {
+  final PageState pageState;
+  final Set<MemberModel> selectedMember;
```

Comment:
do we allow the user to selected more than one member to vouch for?

Reply:
I dont see it on the UI on figma, Just one to be selected

Reply:
fix